keyboards/soda/mango/mango.c

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2015 Álvaro "Gondolindrim" Volpato  <gondolindrim@acheronproject.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "mango.h"
+
+#ifdef RGB_MATRIX_ENABLE
+
+// RGB Matrix 
+led_config_t g_led_config = {
```

Comment:
This should be moved to the info.json config.  You can run `qmk info -k soda/mango -f json` for assistance with that.